utils/vk_concurrent_unordered_map.h

```diff
@@ -95,29 +100,31 @@ typedef std::unique_lock<ReadWriteLock> write_lock_guard_t;
 template <typename Key, typename T, int BUCKETSLOG2 = 2, typename Hash = std::hash<Key>>
 class vk_concurrent_unordered_map {
```

Comment:
When I first brough this over, I changed the name from vl_... to vk_.... That was probably a mistake as it causes unnecessary merge conflicts when bringing other code over.  It would be good to make the name match again and this would be a good time to do it.